core/db_models/EEM_Event_Question_Group.model.php

```diff
@@ -37,4 +38,15 @@ protected function __construct($timezone = null)
         $this->_cap_restriction_generators[ EEM_Base::caps_delete ] = new EE_Restriction_Generator_Event_Related_Protected($path_to_event, EEM_Base::caps_edit);
         parent::__construct($timezone);
     }
+
+    /**
+     * Decides whether to use the 'EQG_primary' or newer 'EQG_additional' for use in queries, based on whether
+     * this is concerning primary attendees or additional attendees.
+     * @since $VID:$
+     * @param $for_primary_attendees
+     * @return string
+     */
+    public function field_name_for_category($for_primary_attendees) {
+        return $for_primary_attendees ? 'EQG_primary' : 'EQG_additional';
```

Comment:
:+1: 
I didn't add `fieldNameForPrimaryRegistration()` though because I didn't see why we'd use that instead of the constant `EEM_Event_Question_Group::PRIMARY`.